Assets/Scripts/GeoObjs/GeoObjDefinitions/Dependent/DependentArc.cs

```diff
@@ -18,10 +18,6 @@ namespace IMRE.HandWaver
 /// </summary>
 	class DependentArc : AbstractArc, DependentFigure
     {
```

Comment:
@nsgnn thanks for flagging this.

We have a weird case where we have a script set up so that we could build out this functionality if we want to, but not prefab to match (or construction in GeoObjConstruction).  That is, Dependent Arcs do not functionally exist.  They will also be obsolete after the GGB/ECS migration.

While this doesn't adhere to best practices in style, we will keep this in the bugfix.